inductiva/tasks/methods.py

```diff
@@ -89,14 +27,104 @@ def get_tasks_info(status: Optional[str] = None,
         }
 
         if status is not None:
-            query_params["status"] = TaskStatusCode(status)
+            query_params["status"] = models.TaskStatusCode(status)
 
         try:
             # Get User Tasks
-            api_response = api_instance.get_user_tasks(
-                query_params=query_params,)
+            resp = api_instance.get_user_tasks(
+                query_params=query_params,
+                skip_deserialization=True,  # avoid deserializing to model,
+                # leave as dict which we'll later serialize to our own
+                # dataclasses
+            ).response
+
+            response_body = json.loads(resp.data.decode("utf-8"))
 
-            return [{**task} for task in api_response.body]
+            return [{**task} for task in response_body]
 
         except ApiException as e:
             raise e
+
+
+# pylint: disable=redefined-builtin
+def list(num_tasks,
+         status: Optional[str | models.TaskStatusCode] = None) -> None:
+    """List the last N tasks of a user.
+
+    This function will fetch info on the last N tasks of a user and print
+    them to stdout formatted as a table. A status can be specified to filter
+    to get only tasks with that status. Tasks are sorted by submission time
+    with the most recent first.
+
+    # pylint: disable=line-too-long
+    Example usage:
+        # list the last 5 tasks that were successful
+        inductiva.tasks.list(5, status="success")
+        ID                   Simulator    Status       Submitted            Started              Duration     VM Type
+        1691150776862178362  openfoam     success      04 Aug, 12:06:17     04 Aug, 12:06:18     0h 1m 53s    c2-standard-8
+        1691149904961476240  openfoam     success      04 Aug, 11:51:46     04 Aug, 11:51:46     0h 1m 28s    c2-standard-8
+        1691081881158823776  openfoam     success      03 Aug, 16:58:02     03 Aug, 16:58:02     0h 1m 20s    n2-standard-32
+        1691081409916619414  openfoam     success      03 Aug, 16:50:11     03 Aug, 16:50:11     0h 1m 20s    n2-standard-32
+        1691080520213617518  openfoam     success      03 Aug, 16:35:21     03 Aug, 16:35:21     0h 1m 23s    n2-standard-32
+    # pylint: enable=line-too-long
+
+    Args:
+        num_tasks: The number of tasks to list.
+        status: The status of the tasks to list. If None, tasks with any status
+            will be listed.
+    """
+    status = models.TaskStatusCode(status) if status is not None else None
+
+    id_header = "ID"
+    simulator_header = "Simulator"
+    status_header = "Status"
+    submitted_header = "Submitted"
+    started_header = "Started"
+    duration_header = "Duration"
+    vm_type_header = "VM Type"
+
+    print(f"  {id_header:<20} {simulator_header:<12} {status_header:<12}"
```

Comment:
This seems very hard-coded and error prone... Shall, we use pandas (we are installing with inductiva anyway)?

Reply:
yup i'll change the way the table is currently being constructed, but  using pandas just to format a table seems a bit overkill